proxy.config.json

```
@@ -77,7 +77,8 @@
       "block": {
         "literals": [],
         "patterns": [],
-        "providers": {}
+        "providers": {},
+        "proxyFileTypes":[]  
```

Suggested name change to keep the configuration property consistent with the rest of the functions (checkSensitiveData, etc).

```suggestion
        "sensitiveFileTypes":[]  
```